apps/src/netsim/NetSimBitLogPanel.html.ejs

```diff
@@ -1,5 +1,5 @@
 <%
-  var i18n = require('./locale');
+  var i18n = require('@cdo/netsim/locale');
```

Comment:
I wonder how much value we get out of having separate imports for separate locales used in the different apps folders. Like maybe we should have something like:

```javascript
import {netsim as i18n} from '@cdo/locale';
```

We could still keep the actual locale files separate, and just import/export them from the main `@cdo/locale` module.